web/src/components/SubscriptionSettings/SubscriptionSettings.js

```diff
@@ -0,0 +1,56 @@
+import { useMutation } from '@redwoodjs/web'
+import { useState } from 'react'
+import { VscLoading } from 'react-icons/Vsc'
+import { useForm } from 'react-hook-form'
+import { Form, Submit } from '@redwoodjs/forms'
+import { navigate, Redirect, routes } from '@redwoodjs/router'
+
+const SUBSCRIBE_USER = gql`
+  mutation createSessionMutation($input: CreateCheckoutSessionInput!) {
+    createCheckoutSession(input: $input)
+  }
+`
+const SubscriptionSettings = () => {
+  const [subscribe, { loading }] = useMutation(SUBSCRIBE_USER)
+  const [subscribed, setSubscribed] = useState('Subscribe Now')
```

Comment:
I feel like we could use a more meaningful name for this state. What about this?

```suggestion
  const [subscriptionStatusMessage, setSubscriptionStatusMessage] = useState('Subscribe Now')
```